ArmPkg/ArmPkg.dec

```diff
@@ -93,6 +93,14 @@
   #
   StandaloneMmMmuLib|Include/Library/StandaloneMmMmuLib.h
 
+  ##  @libraryclass  Provides an interface for a Arm Transfer List.
+  #
+  ArmTransferListLib|Include/Library/ArmTransferListLib.h
+
+  ##  @libraryclass  Provides an interface for a Arm FF-A ABI.
+  #
+  ArmFfaLib|Include/Library/ArmFfaLib.h

```

Comment:
Another general question, with the change pushed here, what is the expected debug print channel? Would the serial log from secure partition have to go through a different UART region? Otherwise, I would expect we should print through the FFA method?

Reply:
Currently, we share same UART-region with TF-A side. because currently, SPMC_AT_EL3 doesn't support related FF-A console interface. ..

Reply:
Does this mean that SPMD, SPMC and secure partitions all print to the same region? I thought we do not allow the same region be shared across different partitions. (i.e. when hafnium parsing the dts for multiple partitions)

Reply:
Currently, the DEBUG print for StandaloneMm is determined Pcd setting for for each PlatformStandaloneMm.dsc.
That's why in according to partition it should be set different region when it builds.

> I thought we do not allow the same region be shared across different partitions. (i.e. when hafnium parsing the dts for multiple partitions)

I'm not sure, for platform the number of UART will be limited, then do you mean the limitation of number of partition should be determined by number of UART ports?

Reply:
If we do not transfer prints from secure partitions to hafnium, then the limitation seems to be tied to number of UART ports?

Reply:
> If we do not transfer prints from secure partitions to hafnium, then the limitation seems to be tied to number of UART ports?

Might be. However, I couldn't find out FFA_CONSOLE_LOG implementation even in hafnum.
Although we send the logs to SPMC, because anyway UART ports is limieted resource, if number of partition is over the number of UART ports, according to pigeonhole principle, some partitions will share it.

If via FFA_CONSOLE_LOG, I think all secure partitions will prints all  log in SPMC's console 
but I'm not use it is right or not :( 

Anyway currently it seems to set the manually via .dsc files by setting "PcdSerialRegisterBase"
which is same to SPMC's console (initilalized by SPMC) to print StandaloneMm's log.